addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -721,3 +721,48 @@ def test_unlink_timesheet(self):
 
         move.with_context(check_move_validity=False).line_ids[0].unlink()
         self.assertFalse(analytic_line.timesheet_invoice_id, "The timesheet should have been unlinked from move")
+
+    def test_non_billable_timesheet(self):
+        """ Test to non-billable timesheet
+
+            1) Create sale order
+            2) Confirm sale_order
+            3) Add the timesheet from the other SOL in the task
```

Comment:
```suggestion
        """ Test non-billable timesheet
        
            Test Cases:
            ==========
            1) Create sale order
            2) Confirm sale_order
            3) Add the timesheet from the other SOL in the task
```
However, I don't understand the test cases for this test. I thought you test a non-billable timesheet but here the timesheet is billable on another SOL than the one you use in the assert.